app/src/main/main.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint global-require: off, no-console: off, promise/always-return: off */
+/* eslint global-require: off, no-console: off, promise/always-return: off, import/first: off */
```

Comment:
disabled `import/first` here to allow properly ordering imports. palette needs to start the profiler before other imports, since they can impact startup performance.

Reply:
LGTM!